distributed/tests/test_client.py

```diff
@@ -5790,22 +5777,14 @@ async def test_config_scheduler_address(s, a, b):
     assert sio.getvalue() == f"Config value `scheduler-address` found: {s.address}\n"
 
 
+@pytest.mark.filterwarnings("ignore:Large object:UserWarning")
 @gen_cluster(client=True)
 async def test_warn_when_submitting_large_values(c, s, a, b):
     with pytest.warns(
         UserWarning,
-        match=r"Large object of size (2\.00 MB|1.91 MiB) detected in task graph:"
-        r" \n  \(b'00000000000000000000000000000000000000000000000 \.\.\. 000000000000',\)"
-        r"\nConsider scattering large objects ahead of time.*",
+        match="Sending large graph of size",
     ):
-        future = c.submit(lambda x: x + 1, b"0" * 2000000)
-
-    with warnings.catch_warnings(record=True) as record:
-        data = b"0" * 2000000
-        for i in range(10):
-            future = c.submit(lambda x, y: x, data, i)
-
-    assert not record
```

Comment:
The specific message "Large object etc." is still issued but on the scheduler. This is not as great as before but still OK. We could refine this by using the client/distributed warn mechanism but I would like to follow up with this in a later PR

Reply:
Fine by me.  Should be easy to do?  Maybe add to a task list for incoming devs.

Reply:
> Maybe add to a task list for incoming devs.

Yes, that was my thinking as well